feature_mining/em_base.py

```diff
@@ -0,0 +1,69 @@
+class ExpectationMaximization:
+    """
+    Base class for Expectation - Maximization algorithm.
+    Treats data preparation, looping, e-step and m-step.
+    """
+
+    def __init__(self, dump_path="../tests/data/em_01/"):
+        print(type(self).__name__, "- base init...")
+
+        # Path used for testing
+        self.dump_path = dump_path
+
+        # Parameters for EM execution control
+        self.max_iter = 50
+        self.lambda_background = 0.7
+        self.dist_threshold = 1e-6
+
+        # Parameters related to collection size
+        self.m = 0
+        self.nw = 0
+        self.na = 0
```

Comment:
Suggest refactoring na to f for "feature"

Reply:
done